openslides/motions/static/js/motions/base.js

```diff
@@ -151,6 +152,21 @@ angular.module('OpenSlidesApp.motions', [
                 getText: function (versionId) {
                     return this.getVersion(versionId).text;
                 },
+                getAgendaItemNumber: function (agendaId) {
```

Comment:
The wording of this method is missleading because it does not return just the number of the respective agenda item but in some cases numbers of ancestor agenda items.